recipes/atomneb/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "atomneb" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/atomneb-{{ version }}.tar.gz
+  sha256: 288cb688aad2fbff919c14a4653931c625ce3431f53ea5257cebe731e6011dbd
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - astropy
+    - numpy
+    - pandas
+  run:
+    - astropy
+    - numpy
+    - pandas
+    - python
```

Comment:
This needs the same version constraint as in host

Reply:
@chrisburr I added the versions in both 'host' and 'run'.